test/e2e_node/pod_host_ips.go

```diff
@@ -276,6 +333,17 @@ func testDownwardAPI(ctx context.Context, f *framework.Framework, podName string
 						},
 					},
 					Env: env,
+					SecurityContext: &v1.SecurityContext{
```

Comment:
do we really need this?
I'm surprised we need this just to create a simple pod, the errors on policy we saw were related to the HostNetwork capability that should be fixed by `	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged`

Reply:
Before adding `admissionapi.LevelPrivileged` according to CI error added before, I try to delete it